ingestion/src/kldata.rs

```diff
@@ -35,47 +44,84 @@ struct ObsinnId {
     sensor_and_level: Option<(i32, i32)>,
 }
 
-fn parse_meta_field<T: FromStr>(field: &str, expected_key: &'static str) -> Result<T, Error>
-where
-    <T as FromStr>::Err: Debug,
-{
-    let (key, value) = field.split_once('=').ok_or_else(|| {
-        Error::Parse(format!(
-            "unexpected field in kldata header format: {}",
-            field
-        ))
-    })?;
+// TODO: maybe this can be a field in ObsinnChunk?
+#[derive(Default)]
+struct ObsinnHeader {
+    station_id: Option<i32>,
+    type_id: Option<i32>,
+    message_id: usize,
+    // TODO: we can parse it to Datatime if we decide we are going to use it
+    received_time: Option<String>,
```

Comment:
This can maybe be &str instead of String? since it's just referencing a substring of the message body?

Reply:
True, I just didn't want to add explicit lifetimes